### DISCUSSION TITLE
Please consider using a different dialog for opening folders.

### DISCUSSION THREAD

**Discussion by herboren** [deepskystacker/DSS#175] [Ideas]
Please consider using CommonOpenFileDialog for browsing and selecting directories, instead of OpenFolderBrowser dialog, The benefit is having a textbox available to quickly navigate directories using the 'Filename' textbox from paths held inside the clipboard for convenience of a quick copy/paste.

https://github.com/aybe/Windows-API-Code-Pack-1.1
![image](https://user-images.githubusercontent.com/987794/224208450-e1b7fd18-c6eb-449c-a3ba-655d07216f75.png)
r

**Comment by perdrix52** [comment]
Windows specific "stuff" is not likely in the Qt based future of DSS

**Reply by herboren** [reply]
I mean thats what its currently using for the FolderBrowserDialog, otherwise whatever is being used looks awfully a lot like the WinForms version right now. Perhaps giving more control than the method used currently in future releases is a good suggestion.

**Comment by tonk777** [comment]
David,

Maybe you should actively disable the issue tracker on Github

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/disabling-issues
[https://github.githubassets.com/images/modules/open_graph/github-logo.png]<https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/disabling-issues>
Disabling issues - GitHub Docs<https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/disabling-issues>
You may wish to turn issues off for your repository if you do not accept contributions or bug reports.
docs.github.com
Cheers, Tony
________________________________
